examples/basic-with-apply/README.md

```diff
@@ -0,0 +1,19 @@
+# Basic with apply button
+
+This example is the same as the [basic example](https://github.com/Lemoncode/react-image-focal-point/tree/main/examples/basic), but it adds a button to apply the focal point changes to the image.
+
```

Comment:
This example is quite similar to the [basic example](https://github.com/Lemoncode/react-image-focal-point/tree/main/examples/basic), but instead of applying the changes real time, the user has to click on an 'apply' button to see the updates reflected

Reply:
Done